Orange/tests/test_txt_reader.py

```diff
@@ -48,29 +37,25 @@ def read_easy(self, s, name):
         try:
             file.write(s)
             file.close()
-            table = TxtFormat().read_file(filename)
+            table = CSVFormat().read_file(filename)
 
             f1, f2, f3 = table.domain.variables
-            self.assertIsInstance(f1, ContinuousVariable)
-            self.assertEqual(f1.name, name + "1")
+            self.assertIsInstance(f1, DiscreteVariable)
```

Comment:
Yes, the new rule, in https://github.com/kernc/orange3/commit/76303e934b393b111b941844c135014bd6b5069a, is `all(int(i) == i) → discrete`. @janezd gave verbally his hesitant but unconditional agreement on that.